pkg/apis/core/validation/validation_patch.go

```diff
@@ -39,6 +39,7 @@ var (
 		"openshift-kube-apiserver-operator":          {},
 		"openshift-kube-apiserver":                   {},
 		"openshift-kube-controller-manager-operator": {},
+		"openshift-config-managed":                   {},
```

Comment:
Do we have a list somewhere that includes all the affected secret names that we currently know about?